### PR TITLE
Update to version 1.0.3

### PR DESCRIPTION
**Pull request by hmaarrfk**
I also modernized the script

### CODE REVIEW

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by hmaarrfk** [comment]
@conda-forge-admin, please rerender

**Comment by hmaarrfk** [comment]
@jwiggins just a friendly ping about updating this package on conda-forge. If you would like, I could add myself as a maintainer to take some load off you.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,32 +7,28 @@ package:
...
-    - freetype 2.6.*
+    - freetype >=2.6
   run:
     - python
-    - vc 9  # [win and py27]
-    - vc 14  # [win and py>=35]
-    - numpy >=1.11
-    - freetype 2.6.*
+    - {{ pin_compatible('numpy') }}
+    - freetype >=2.6
```
Comment (ocefpaf) [inline recipe/meta.yaml]: You can remove this pin and let the re-rendering take care of that.

Comment (hmaarrfk) [reply recipe/meta.yaml]: freetype?

Comment (ocefpaf) [reply recipe/meta.yaml]: Yep. On `host` and `run`.

**Comment by jwiggins** [comment]
@hmaarrfk Yes, I would appreciate the help.

(I'm now watching this repo. Sorry for not responding earlier)

**Comment by jwiggins** [comment]
So we're basically waiting for fixes to happen upstream before this can be merged?

**Comment by ocefpaf** [comment]
> So we're basically waiting for fixes to happen upstream before this can be merged?

If you want to "cash" in the `OS X` and `Linux` packages you can go ahead and merge this. Just open an issue to tracker the `Windows` problem there.

**Comment by jwiggins** [comment]
I'm not in any particular hurry

**Comment by hmaarrfk** [comment]
neither am I really, I got good at rebasing and rerending in case conda-smithy changes.

**Comment by hmaarrfk** [comment]
@ocefpaf any clue why this error is showing up https://ci.appveyor.com/project/conda-forge/celiagg-feedstock/build/1.0.19/job/j43k1cruydyeccxy#L632

**Comment by ocefpaf** [comment]
> @ocefpaf any clue why this error is showing up ci.appveyor.com/project/conda-forge/celiagg-feedstock/build/1.0.19/job/j43k1cruydyeccxy#L632

No idea. Sorry... Windows is not my thing :-/

**Comment by hmaarrfk** [comment]
@jwiggins I guess Windows builds are also new for celiagg, but any idea why the following happens:

@isuruf @ 00:27 on gitter said:
```
It's a weird issue.
pip + numpy 1.12 works
pip + numpy 1.11 doesn't
python setup.py + numpy 1.12 works
python setup.py + numpy 1.11 works
```

**Comment by jwiggins** [comment]
Looks like it passed! (🎊)

Does 3.5 support _need_ to be dropped? What do you mean by "pinning 2018.10.03" in your commit message?

**Comment by jwiggins** [comment]
And to answer your question, no, I don't know why the recipe fails for pip + numpy 1.11. I only remember having trouble with Python 3.4 back when I last worked on this recipe.

**Comment by hmaarrfk** [comment]
@jwiggins I think we can revert to an older version of conda-smithy and conda-forge-pinning and rerender the recipe.

**Comment by hmaarrfk** [comment]
@jwiggins this should build for python 3.5. We can rerender again after that.

**Comment by hmaarrfk** [comment]
@jwiggins friendly ping. I would merge this before something crazy happens with Travis making 3.5 compilation impossible again or the bot for 3.7 migration comes around.

**Comment by jwiggins** [comment]
The build was clean(-ish. Had to restart linux3.5): https://anaconda.org/conda-forge/celiagg/files?version=1.0.3

**Pull request merged**